data/akshaykamath45.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Akshay Kamath",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Learning  **Web Development**.Computer Engineering Student",
+  "avatar":"https://github.com/akshaykamath45.png",
```

Comment:
We no longer need the avatar key anymore. It will auto-pick your GitHub profile, so please remove this field! 
You can refer the updated docs: https://linkfree.eddiehub.io/docs/quickstart

```suggestion
```

Reply:
I have directly used the `Commit suggestion`,is it fine?

Reply:
Can I mark it as `Resolve conversation`

Reply:
> Can I mark it as `Resolve conversation`

Yeah, sure! Go ahead 👍